doc/source/getting_started/running_mapdl.rst

```diff
@@ -197,18 +254,175 @@ For Linux:
 
 Note that you should probably startup MAPDL in a temporary working
 directory as MAPDL creates a several temporary files.
+You can specify a directory by either launching MAPDL from the temporary directory:
+
+.. code:: pwsh
+
+    mkdir temporary_directory
+    cd temporary_directory
+     & 'C:\Program Files\ANSYS Inc\v222\ansys\bin\winx64\ANSYS222.exe'
+
+Or specifying the ``-dir`` flag:
+
+.. code:: pwsh
+
+    mkdir temporary_directory
+    & 'C:\Program Files\ANSYS Inc\v222\ansys\bin\winx64\ANSYS222.exe' -dir "C:\ansys_job\mytest1"
+
+
+If this command doesn't launch, please pay attention to the command output:
+
+.. code:: pwsh
+
+    (base) PS C:\Users\user\temp> & 'C:\Program Files\ANSYS Inc\v222\ansys\bin\winx64\ANSYS222.exe'
+    *** ERROR ***
+    Another ANSYS job with the same job name (file) is already running in this
+    directory or the file.lock file has not been deleted from an abnormally
+    terminated ANSYS run.  To disable this check, set the ANSYS_LOCK environment
+    variable to OFF.
+
+
+There is a variety of issues that can make ANSYS not launching, including:
+
+- License server setup.
+  Incorrect license server configuration can make ANSYS not able to get a valid license.
+  In those cases, you might see output **similar** to:
+
+  .. code:: pwsh
+
+    (base) PS C:\Users\user\temp> & 'C:\Program Files\ANSYS Inc\v222\ansys\bin\winx64\ANSYS222.exe'
+   
+    ANSYS LICENSE MANAGER ERROR:
+
+    Maximum licensed number of demo users already reached.
```

Comment:
This should be a a complete sentence: Maximum licensed number of demo users has already been reached.

Reply:
That is the output of MAPDL. We cannot change it :(